### ISSUE TITLE
Cloud Storage

### ISSUE THREAD

**Issue by Lykrast** [EnigmaticaModpacks/Enigmatica9#14]
### URL

https://www.curseforge.com/minecraft/mc-mods/alexs-cloud-storage

### Why would you like the mod added?

This is essentially a duplicate of my accepted E8 suggestion https://github.com/EnigmaticaModpacks/Enigmatica8/issues/127 since I can't find a visible modlist on here yet and this cool mod was ported and I'm supposing E9 is to replace E8 if I remember that blog post right... anyways I'm gonna copy the points:

* Basically modded Ender Chests with extra flavor
* New mod so gives it some spotlight *(though Alex is not an obscure modder)*
* A few small but neat structures in the sky to loot, with the challenge of being very high in the sky with little footing
* Extra pet for Domestication Innovation **(hey it got ported too!)**
* I like it

**Comment by MuteTiefling** [comment]
Had a play around with this. seems neat as a mobile storage, but doesn't appear to interface with anything? Maybe I'm missing something, but it doesn't appear that you can pipe in/out of it. Doesn't interface with Tom's Simple Storage either. 

I suppose these can be pros or cons depending how we look at it :D

**Comment by Lykrast** [comment]
> Had a play around with this. seems neat as a mobile storage, but doesn't appear to interface with anything? Maybe I'm missing something, but it doesn't appear that you can pipe in/out of it. Doesn't interface with Tom's Simple Storage either.

That seemed weird to me cause I remember reading it like the modded Ender Chests with the color combinations, so I looked back at the mod page and found this:

> **Cloud chests themselves are private storage** - like an ender chest, **they are exclusive to you and cannot be opened by other players, or accessed by hoppers**. Luckily, if you need something like that, there is **the static cloud chest**, made in the same way as the cloud chest but with static cloud blocks. **These can be accessed by hoppers, pipes, other players, etc.** as they are static and the same for everyone. To upload to the static inventory, you will need a statically-charged balloon, which you can get by rubbing a balloon against a static cloud.

**Comment by MuteTiefling** [comment]
Hrm, ok still appears to be wonky. 

https://github.com/Alex-the-666/CloudStorage/issues/32

The Static Chest is also not being seen by Tom's. I currently can't see this as anything more than a very large personal ender chest, however. Hopefully the automation side will come along though.

**Comment by NielsPilgaard** [comment]
Neat mod, but due to the issue above and the lack of responses on the issue tracker it's gonna be a no for the time being.
